pulp_ansible/app/viewsets.py

```diff
@@ -288,6 +319,40 @@ def sync(self, request, pk):
         )
         return OperationPostponedResponse(result, request)
 
+    @extend_schema(
+        description="Trigger an asynchronous task to sync Ansible content.",
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    @action(detail=True, methods=["post"], serializer_class=AnsibleRepositorySignatureSerializer)
+    def sign(self, request, pk):
```

Comment:
I like this design, to sign everything and then make a new repo version out of it.